python/letsql/expr/api.py

```diff
@@ -1709,3 +1712,7 @@ def to_parquet(
         with pq.ParquetWriter(path, batch_reader.schema, **kwargs) as writer:
             for batch in batch_reader:
                 writer.write_batch(batch)
+
+
+def train_test_splits(*args, **kwargs) -> Iterator[tuple[ir.Table, ir.Table]]:
+    return ml.train_test_splits(*args, **kwargs)
```

Comment:
when we do this, we lose the signature
can we use
`from letsql.expr.ml import train_tests_splits`
instead?

Reply:
@ghoersti why do we redefine `train_test_splits` in `api.py` rather than importing from `ml.py`?

Reply:
this is the impact of redefining with `*args, **kwargs` rather than importing
```
In [1]: import letsql as ls

In [2]: ls.train_test_splits
Out[2]: <function letsql.expr.api.train_test_splits(*args, **kwargs) -> 'Iterator[tuple[ir.Table, ir.Table]]'>

In [3]: ls.expr.ml._train_test_splits
Out[3]: <function letsql.expr.ml._train_test_splits(table: ibis.expr.types.relations.Table, unique_key: str | list[str], test_sizes: Union[Iterable[float], float], num_buckets: int = 10000, random_seed: int | None = None) -> Iterator[ibis.expr.types.relations.Table]>
```